src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue/Infrastructure/Data/Repositories/DatalockRepository.cs

```diff
@@ -18,7 +17,7 @@ public DatalockRepository(string connectionString) : base(connectionString)
         {
         }
 
-        public List<DatalockOutputEntity> GetDatalockOutputForProvider(long ukprn, DateTime firstDayOfNextAcademicYear)
+        public List<DatalockOutputEntity> GetDatalockOutputForProvider(long ukprn)
         {
             const string sql = @"
```

Comment:
as the sql scripts are getting more involved, I suggest moving them to text files in the solution that are then referenced in code using a resource.

Reply:
Probably need to talk about this one. Not sure what you're after. My view is that the SQL scripts are actually getting simpler

Reply:
not so much for this pr tbh.